### DISCUSSION TITLE
Achievements not getting updated

### DISCUSSION THREAD

**Discussion by krishvoor** [community/community#71549] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hey, I believe I  https://github.com/krishvoor submitted more than 128 PR's.
Am I eligible for Pull Shark x 3?

**Comment by krishvoor** [comment]
Can someone update on this?

**Comment by ivandda** [comment]
Yes, you already have pull shark x3. Congratulations!

**Comment by SuperZekes** [comment, marked as answer]
You have it on your profile but if you ever dont see it make sure your badges are not hidden. You can check under public profile in settings
![Screenshot 2023-11-11 051456](https://github.com/community/community/assets/149198602/9a6f7b6b-c810-4353-bc79-f0fc1b3128d7)

**Comment by krishvoor** [comment]
I see them now - thanks for your responses.

**Reply by SuperZekes** [reply]
glad I could help :D
